first_ruby/bin/setup

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
```

Comment:
1. Whats this file for?
2. How do you use this file?

Reply:
1. This is for running an existing project in different machine and which will use for setup the project environment. 
2. By running `ruby bin/setup`, example of setup is set up database, set up ruby dependencies etc.